src/applications/rated-disabilities/components/RatedDisabilityListItem.jsx

```diff
@@ -17,7 +17,7 @@ const RatedDisabilityListItem = ({ ratedDisability }) => {
   const headingText = getHeadingText(ratedDisability);
 
   return (
-    <va-card class="vads-u-margin-bottom--2">
+    <va-card class="vads-u-margin-bottom--2" uswds="false">
```

Comment:
I guess `va-card` doesn't have a `uswds` property either - I've never used it, so I didn't know 😝